src/main/resources/application.yaml

```diff
@@ -1,10 +1,6 @@
 server:
   port: 8485
 
-management:
-  security:
-    enabled: ${MANAGEMENT_SECURITY_ENABLED:true}
```

Comment:
is there are different config for this now?

Reply:
`management.security.*` has been completely removed and only can be editable via java code. since we not even using the spring boot security package - i don't think this config was even applied at any point in the past